tt_eager/tt_dnn/op_library/ccl/reduce_scatter/reduce_scatter_op.cpp

```diff
@@ -65,14 +65,14 @@ std::vector<Tensor> reduce_scatter_impl(
     const uint32_t scatter_dim,
     const uint32_t num_links,
     const MemoryConfig& output_mem_config,
-    const ccl::Topology topology) {
+    const ttnn::utils::ccl::Topology topology) {
     TT_FATAL(std::getenv("TT_METAL_SLOW_DISPATCH_MODE") == nullptr, "This op is only supported for Fast Dispatch");
 
     std::vector<Tensor> output_tensors;
     output_tensors.reserve(input_tensors.size());
     std::vector<ReduceScatter> ops;
     ops.reserve(input_tensors.size());
-    bool is_ring = topology == ccl::Topology::Ring;
+    bool is_ring = topology ==ttnn::utils::ccl::Topology::Ring;
```

Comment:
not clear why this is in utils namespace.. looks like it would be better to just remove the utils namespace and keep things in `ttnn::ccl::..`